docs/usage/sc_AIRRseq/single_cell_tutorial.md

```diff
@@ -0,0 +1,156 @@
+
+
+## Understanding the results
+
+After running the pipeline, several reports are generated under the result folder. 
+
+![example of result folder](tutorial_images/airrflow_result_folder_example.png)
+
+
+The summary report, named 'Airrflow_report.html', provides an overview of the analysis results, such as the V(D)J gene assignment and QC, and V gene family usage. Additionally, it contains links to detailed reports for other specific analysis steps. 
+
+The analysis steps and their corresponding folders, where the results are stored, are listed below. 
+
+1. QC and sequence assembly (if starting from fastq files). 
+   - In this first step, Cell Ranger's VDJ algorithm is employed to assemble contigs, annotate contigs, call cells and generate clonoytpes. The results are stored in the 'cellranger' folder.  
+
+2. V(D)J annotation and filtering. 
+   - In this step, gene segments are assigned using a germline reference. Alignments are annotated in AIRR format. Non-productive sequences and sequences with low alignment quality are removed. Metadata is added. The results are under the folder named 'vdj_annotation'. 
+
+3. QC filtering. 
+   - In this step, cells without heavy chains or with multiple heavy chains are removed. Sequences in different samples that share the same cell_id and necleotide sequence are filtered out. The result are stored in the 'qc-filtering' folder. 
+
+4. Clonal analysis. 
+   - In this step, the Hamming distance threshold of the junction regions is determined when clonal_threshold is set to 'auto' (by default).it should be reviewed for accuracy once the result is out. The threshold result can be found under the folder clonal_analysis/find_threshold. 
+   - If the automatic threshold is unsatisfactory, you can set the threshold manually and re-run the pipeline. 
+   (Tip: use -resume whenever running the Nextflow pipeline to avoid duplicating previous work). 
+   - For TCR data, where somatic hypermutation does not occur, set the clonal_threshold to 0 when running the Airrflow pipeline.  
+   - Once the threshold is established, clones are assigned to the sequences. A variety of tables and plots associated with clonal analysis were added to the folder 'clonal_analysis/define_clones', such as  sequences_per_locus_table, sequences_per_c_call_table, sequences_per_constant_region_table,num_clones_table, clone_sizes_table,clone size distribution plot, clonal abundance plot, diversity plot and etc. 
+
+5. Repertoire analysis. 
+   - The output folder is'repertoire_comparison'. V gene distribution tables and plots are included in this folder.
+
+6. Other reporting.
+   - Additional reports are also generated, including: a multiqc report which summarizes QC metrics across all samples, pipeline_info reports and report_file_size reports.
+
```

Comment:
Could you rewrite this part a bit focusing on the different html reports and a general description on what is inside them?